code_design/guard_clauses.md

```diff
@@ -0,0 +1,27 @@
+# Replace if/else conditionals with guard clauses
+
+When doing a lot of nested conditionals you may end facing a code like this:
+
+```ruby
+def result_with_nested_conditionals
+  if condition_one
+    return result_one
+  elsif condition_two
+    return result_two
+  else
+    return result_three
+  end
```

Comment:
Yeah, perhaps the phrasing you're looking for is a conditional with many *branches*?